_sass/module/_hero.scss

```diff
@@ -42,145 +38,118 @@ $hero-breakpoint: $grid--medium;
     color: $color-white;
     text-shadow: 0 2px 3px $text-shadow-color;
   }
-
-  // Include a scrim over photo backgrounds to increase legibility of content
-  &::before {
-    content: '';
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    right: 0;
-  }
 }
 
-// Layout element
-.hero__header {
-  @include grid-column(12);
-  text-align: center;
+.hero__content {
+  max-width: 120rem;
 }
 
-// Layout element
-.hero__main {
-  @include grid-column(12);
-  font-weight: 350;
-  font-style: italic;
+.hero__header {
   display: flex;
+  flex-direction: column;
   justify-content: center;
-}
-
-.hero__hed {
-  font-size: $title-font-size;
-  font-weight: 375;
+  width: 100%;
+  padding: 5rem;
 
   @include media($hero-breakpoint) {
-    font-style: italic;
+    padding: 0;
+    @include grid-column(1, $custom-grid--thirds);
   }
-}
 
-.hero__subhead {
-  @include heading('h3');
-  color: $color-white;
-  font-style: italic;
+  .btn {
+    background-color: $color-white;
+    font-style: normal;
+    font-weight: bold;
+    width: 90%;
+  }
 }
 
-.hero__lede {
-  text-align: center;
+.hero__subheader {
+  font-style: italic;
 
   @include media($hero-breakpoint) {
-    text-align: initial;
+    margin-top: $spacing-base * 7;
   }
 }
 
-.hero__content {
-  @extend .grid--narrow;
-  position: relative; // layer the content above the .hero--photo scrim
-}
-
-.hero__section {
-  padding-bottom: $spacing-base * 2;
-}
-
-.hero__cta-container {
-  @include grid-collapse;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+.hero__hed {
+  font-size: $h2-font-size;
```

Comment:
Compared to the design, the header title text ("Track the money in Oakland elections") looks to be a heavier font weight, and it looks kinda cramped.
I don't know all the font size/weight options off the top of my head, but maybe we could thin this down slightly to more closely match the design and give it some more breathing room.

Reply:
Oops, this might have been taken care of by Colin's [commit](https://github.com/caciviclab/odca-jekyll/pull/348/commits/202438d8034c5ab1fa1d52342ad09d834c941150). I was just looking at the screenshots, but they might be _slightly_ outdated.